src/main/java/me/coley/recaf/command/impl/Export.java

```diff
@@ -162,6 +165,8 @@ public static void writeArchive(File output, Map<String, byte[]> content) throws
 				jos.closeEntry();
 			}
 		}
+		Files.write(output.toPath(), bytes.toByteArray(),
+				StandardOpenOption.CREATE, StandardOpenOption.TRUNCATE_EXISTING);
```

Comment:
Looks like the options here do not need to be specified: https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#write(java.nio.file.Path,%20byte[],%20java.nio.file.OpenOption...)

> If no options are present then this method works as if the CREATE, TRUNCATE_EXISTING, and WRITE options are present.